surveys/models.py

```diff
@@ -39,6 +38,11 @@ class SurveyFormEntry(FormEntry):
         default='intercept'
     )
 
+    active = models.BooleanField(default=True)
```

Comment:
This attribute looks good! I'll note that it's a common convention, in Python and elsewhere, to use predicate clauses to name boolean variables and attributes (e.g. `active` would be `is_active`). See this discussion on SE: https://softwareengineering.stackexchange.com/a/221873

This is a common convention related to style and it's not enshrined in any of our best-practices documents, so don't worry about changing your attribute name. I only wanted to point out the convention for your consideration.